File: AdvancedMapAndroid/app/src/main/java/com/carto/advancedmap/model/Sample.java
package com.carto.advancedmap.model;

/**
 * Created by aareundo on 21/09/2017.
 */

public class Sample {

    public int imageResource;

    public String title, description;

    public Class<?> activity;

    public Sample(int resource, String title, String description, Class activity) {
        this.imageResource = resource;
        this.title = title;
        this.description = description;
        this.activity = activity;
    }
}

File: AdvancedMapKotlin/app/src/main/java/com/carto/advanced/kotlin/model/Sample.kt
package com.carto.advanced.kotlin.model

/**
 * Created by aareundo on 13/07/2017.
 */

class Sample() {

    var imageResource: Int? = null

    var title: String? = null

    var description: String? = null

    var activity: Class<*>? = null

    init {

    }

    constructor(imageResource: Int, title: String, description: String, activity: Class<*>) : this() {
        this.imageResource = imageResource
        this.title = title
        this.description = description
        this.activity = activity
    }
}